### DISCUSSION TITLE
Savings bank and loan accounts add money to "my balance", but not "my savings"

### DISCUSSION THREAD

**Discussion by plant7403** [firefly-iii/firefly-iii#5604] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely me, not you](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

Hello, i'm want to invest some money and i have some problems with this
First i want to put the money on my savings account, then move it to "loan owed"
And what i see my "savings bank" is displaying the money as balance, same with the loan account

### Expected behaviour

Usually savings account is for savings, so i think it should display them the same way
Same with loans or investments
Debts do it the right way, when i create a debt of $1 it shows -1$ in the savings

### Debug information

Debug information generated at 2022-01-23 20:42:02 Europe/Moscow for Firefly III version **5.6.10**.

| Scope | Version |
| --- | --- |
| Firefly III | 5.6.10 |
| Firefly III API | 1.5.5 |
| Build | 75, 09-01-2022 08:36:35 CET |
| PHP | 8.0.13 |
| Host | Linux |

| System info | Value |
| --- | --- |
| Installation ID | 868f5970-c35a-4d5f-b674-047263eeaef7 |
| Using docker? | true |
| Layout | v1 |
| System TZ | Europe/Moscow |
| Browser TZ | Asia/Ho_Chi_Minh |
| App environment | production |
| App debug mode | false |
| App cache driver | file |
| App logging | notice, stack |
| Display errors | Off |
| Error reporting | ALL errors |
| Interface | apache2handler |
| Default language | en_US |
| Default locale | equal |
| BCscale | 24 |
| DB drivers | mysql, pgsql, sqlite |
| Current driver | mysql |
| DB version | 18 (exp. 18) |
| Login provider | eloquent  |
| Trusted proxies (.env) | ** |

| User info | Value |
| --- | --- |
| Session start | 2022-01-01 00:00:00 |
| Session end | 2022-01-31 23:59:59 |
| Session first | 2022-01-23 00:00:00 |
| User ID | 1 |
| User language | ru_RU |
| User locale | ru_RU |
| Attempt at "ru_RU.utf8" | 'ru_RU.utf8' |
| Attempt at "ru_RU.UTF-8" | 'ru_RU.UTF-8' |
| User agent | Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:91.0) Gecko/20100101 Firefox/91.0 Waterfox/91.2.0 |

### Steps to reproduce

1. Put money on your Bank Savings
2. See the Balance has that money

or

1. Put money on somebody's loan, even from Bank Savings
2. See the Balance has that money

**Comment by JC5** [comment]
I don't understand what you're trying to do.

**Comment by plant7403** [comment]
Some examples:
I want to borrow some money to my friend
I want to put money in a bank to get a %
I want to invest money in stock market

How would i do this? I want all of these to be in my net worth, but not in balance
Also, is interest rate working for investing? And how to make it show the future aacount balance with included interest

**Comment by JC5** [comment]
- The balance field updates every month, don't bother with it. It's just all the amounts added up. Ignore it, it will reset next month
- It's not. Add the amount manually or ignore it.

**Comment by plant7403** [comment]
I'm confused...
So i can't borrow money to somebody without it showing in my balance? 🥲
Also, why my savings account is showing only in balance but not in net worth? How would i add money to net worth then?
(And i just noticed, interest rate is not calculated in any way, i thought it should🥲)

**Comment by plant7403** [comment]
And also it's impossible to add interest rate for bank accounts? 🥲

**Comment by plant7403** [comment]
Also it seems the things i wanted to check don't work i demo, it doesnct even let me transfer funds to any loan/debt
